docs/source/reference/image_classification.rst

```diff
@@ -61,6 +61,8 @@ Here's the full example:
     :lines: 14-
 
 To learn how to view the available backbones / heads for this task, see :ref:`backbones_heads`.
+Moreover the backbone benchmarks of TIMM models can be found here:
```

Comment:
Should we instead use `PyTorch Image Models` here? I was confused about what's TIMM.

```suggestion
Moreover, the benchmarks of different backbones of PyTorch Image Models (TIMM) can be found here:
```